upstream-community-operators/storageos/storageos.v0.0.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: storageosoperator.v0.0.10
+  namespace: placeholder
+  annotations:
+    capabilities: Full Lifecycle
+    categories: "Storage"
+    alm-examples: '[{"apiVersion":"storageos.com/v1alpha1","kind":"StorageOSCluster","metadata":{"name":"example-storageos","namespace":"default"},"spec":{"secretRefName":"storageos-api","secretRefNamespace":"default"}},{"apiVersion":"storageos.com/v1alpha1","kind":"Job","metadata":{"name":"example-job","namespace":"default"},"spec":{"image":"storageos/cluster-operator:latest","args":["/var/lib/storageos"],"mountPath":"/var/lib","hostPath":"/var/lib","completionWord":"done"}},{"apiVersion":"storageos.com/v1alpha1","kind":"StorageOSUpgrade","metadata":{"name":"example-upgrade","namespace":"default"},"spec":{"newImage":"storageos/node:latest"}}]'
+    description: Cloud-native, persistent storage for containers.
+spec:
+  # replaces: 
+  displayName: Storageos
```

Comment:
Should this be `StorageOS` ?

Reply:
oops! Right, that's needs to be fixed. Thanks!